multiview/cluster/mv_k_means.py

```diff
@@ -0,0 +1,262 @@
+import numpy as np
+from .base_cluster import BaseCluster
+from ..utils.cluster_utils import check_Xs
+
+
+class MultiviewKMeans(BaseCluster):
+
+    '''
+    An implementation of Multi-View K-Means using the co-EM algorithm.
+    This algorithm currently handles two views of data.
+
+    Paramters
+    ---------
+    k : int
+        The number of clusters
+
+    random_state : int (default=None)
+        Determines random number generation for initializing centroids.
+        Can seed the random number generator with an int.
+
+    max_iter : int (default=None)
```

Comment:
This doesn't seem to a class parameter? Looks like it's only called on the function level. I would remove here. Otherwise, I would set max_iter in the constructor and remove the parameter from the functions (this is what sklearn does: https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/cluster/_k_means.py so might be preferred)